src/extension/background.js

```diff
@@ -66,10 +66,16 @@ async function getConfigFromTabUrl(tabUrl) {
   const share = getShareSettings(tabUrl);
   if (share.giturl) {
     // share url
-    return getGitHubSettings(share.giturl);
+    return {
+      ...share,
```

Comment:
@icaraps I think the `giturl` should be removed from `share` here.

Reply:
Oh why ? Is the config malformed ? When making these changes, I wasn't sure how the config object should look like:
> @returns {object} The config object

Reply:
Just a cleanliness issue. It is no longer used downstream.

Reply:
Ok I undefined its value.